CONTRIBUTING.md

```diff
@@ -17,4 +17,43 @@ This project and everyone participating in it is governed by the code of conduct
 ## How can I contribute?
 1. You can report bugs/glitches/faults that you are likely to come across when using the project server and client apps.
 2. You can suggest enhancements that you would want to see implemented on the project - which gets voted in the roadmap.
-3. You can address a wide variety of open issues if you wish to contribute using your code and make a pull request.
\ No newline at end of file
+3. You can address a wide variety of open issues if you wish to contribute using your code and make a pull request.
+
+## Steps to contribute
+
+* Comment on the issue you want to work on. Make sure it's not assigned to someone else.
+
+* If you think you encounter a bug or have a suggestion for improvement of code or to add a feature, then create a issue with proper title and tags but first make sure it's not already present. And if you want to work on that, then comment on that issue and wait for the owner to assign that work to you.
+
+### Making a PR
+
+> - Make sure you have been assigned the issue to which you are making a PR.
+> - If you make PR before being assigned, It will be labeled `invalid` and closed without merging.
+
+* Fork the repo. This will create a copy of the current repository that you can edit and make changes to.
+
+* Clone it on your machine. But first navigate to your forked repo and then, there you will find link provided by github to clone.
+
+* Add a upstream link to main branch in your cloned repo
+    ```
+    git remote add upstream https://github.com/t0xic0der/sanctuary-zero.git
+    ```
+* Keep your cloned repo upto date by pulling from upstream (this will also avoid any merge conflicts while committing new changes)
+    ```
+    git pull upstream main https://github.com/t0xic0der/sanctuary-zero.git 
+    ```
+* Create your feature branch
+    ```
+    git checkout -b <feature-name>
+    ```
+* Commit all the changes
+    ```
+    git commit -m "Meaningful commit message"
+    ```
+* Push the changes for review
+    ```
+    git push origin <branch-name>
+    ```
+* Create a PR from your repo on Github. Give proper title and description of the changes you made.
```

Comment:
Change `Github` to `GitHub`.

Reply:
Change `Give proper title and description of the changes you made.` to `Give a proper title and description for the changes you made.`